### ISSUE TITLE
Fences and elevation contours look the same

### ISSUE THREAD

**Issue by fancsali** [freizeitkarte/fzk-mde-garmin#30]
First of all, your map is awesome, one of the best, if not the best hiking map I've found for Garmin devices so far.
However, I got myself in a bit of a pickle earlier today: I was walking downhill in a bushy area, crossing lots of elevation contours, when I suddenly found one line I couldn't cross, as it was actually a fence. (Confirmed by moving the pointer above it.)

I reckon, the problem is, they are both rendered as grey lines (Confirmed on my eTrex Vist HCx, and my computer loading the maps in qMapShack) I believe, it'd be beneficial to adjust the style slightly, and make those contours brown, as it is customary.

**Comment by teddyimsuff** [comment]
This is not an issue with the map. It's an issue with the design. Look into using other TYP files for your map.

**Comment by fancsali** [comment]
Indeed, that's an issue with the TYP files. However, that's exatcly what the 'Freizeitkarte' project is about: taking OSM data, providing TYP files, and turning it into usable maps on Garmin devices.

Hence I reckon, my above comment is still applicable here...

**Comment by teddyimsuff** [comment]
What Typ did you use?
My solution was to change the design of the Polyline representing the fence and not to rely on colour alone 

Do you need any help with that?
